### DISCUSSION TITLE
Layout issue with Tailwind 3

### DISCUSSION THREAD

**Discussion by philippeadam** [solspace/craft-freeform#1419] [Questions & Answers]
### What happened?

When using the default layout feature with Tailwind 3, the resulted layout doesn't match the displayed layout in the CMS.

CMS:
![Screenshot 2024-07-11 at 10 55 16 AM](https://github.com/solspace/craft-freeform/assets/48809185/e6a2b5c6-1822-465a-b67e-5aece4d9b0b1)
![Screenshot 2024-07-11 at 10 58 45 AM](https://github.com/solspace/craft-freeform/assets/48809185/58b9d2a3-288c-4c63-a686-617f85ff9157)

Result:
![Screenshot 2024-07-11 at 10 56 30 AM](https://github.com/solspace/craft-freeform/assets/48809185/10af6846-82d5-4a60-8498-37c8be88715b)

If I remove the flex-wrap in the inspect element it seems to fix the problem:
![Screenshot 2024-07-11 at 10 57 25 AM](https://github.com/solspace/craft-freeform/assets/48809185/be869d65-7280-42d6-b6b7-44facfd106d3)
![Screenshot 2024-07-11 at 10 59 33 AM](https://github.com/solspace/craft-freeform/assets/48809185/d81e0184-c467-4cff-a14b-1921de80d83b)

### How can we reproduce this?

1. Use Tailwind 3.4.3 
2. Use Freeform 5.3.0
3. Create a form and put 2 fields side by side

### Freeform Edition

Pro

### Freeform Version

5.3.1

### Craft Version

5.1.6

### When did this issue start?

Fresh install of Freeform

**Comment by kjmartens** [comment]
Hi @philippeadam,

I am not able to reproduce this in my testing. I am wondering if you can try and confirm a few things:

1. Update to Freeform 5.5.4 and see if that makes any difference.
2. Install the demo templates included with Freeform and see if the form appears correctly there **_OR_**
3. Create a brand new template with nothing else on it except something like this:
```
<html>
<head></head>
<body>

{{ freeform.form("yourFormHandle", {
    formattingTemplate: 'tailwind-3/index.twig'
}).render() }}

</body>
</html>
```
4. Is there any chance you have custom CSS overrides that could be interfering with the form?

Thanks 🙂

**Comment by philippeadam** [comment]
@kjmartens I have upgraded to 5.5.4 and also tried placing only the form on an empty page but it still doesn't put the fields side by side. 
![Screenshot 2024-07-12 at 1 55 48 PM](https://github.com/user-attachments/assets/e0e1d9af-4dd9-42cf-8ea1-3a35a72102f4)
![Screenshot 2024-07-12 at 1 56 36 PM](https://github.com/user-attachments/assets/cbf4b38e-6af7-47e7-ac9e-b979a9da4155)
